src/pickers/ClaimOfficerPicker.js

```diff
@@ -21,47 +23,50 @@ const ClaimOfficerPicker = (props) => {
   const modulesManager = useModulesManager();
   const { formatMessage } = useTranslations("claim", modulesManager);
   const [variables, setVariables] = useState({});
+  const dispatch = useDispatch();
 
-  const { isLoading, data, error } = useGraphqlQuery(
-    `
-    query ClaimOfficerPicker ($search: String) {
-      claimOfficers(search: $search, first: 20) {
-        edges {
-          node {
-            id
-            uuid
-            code
-            lastName
-            otherNames
-            ${extraFragment ?? ""}
-          }
-        }
-      }
-    }
-  `,
-    variables,
-    { skip: true },
-  );
+  const isLoading = useSelector((state) => state.claim.claimOfficers.isFetching);
+  const isLoaded = useSelector((state) => state.claim.claimOfficers.isFetched);
+  const options = useSelector((state) => state.claim.claimOfficers.items);
+  const error = useSelector((state) => state.claim.claimOfficers.error);
+
+  useEffect(async () => {
+     await dispatch(
+      fetchClaimOfficers(modulesManager, extraFragment, variables, {
```

Comment:
Number of parameters in this function call doesn't match fetchClaimOfficers definition (3 arguments in function definition and 4 in here). It will not result in any runtime error but has to be fixed. 
Also I don't think you need `{first:20}` part at all as it's already included in GQL Query string in `fetchClaimOfficers` function.